pymc/distributions/distribution.py

```diff
@@ -979,36 +968,36 @@ def __new__(
         dist_params = cls.parse_dist_params(dist_params)
         cls.check_valid_dist_random(dist, random, dist_params)
         if dist is not None:
+            kwargs.setdefault("class_name", f"CustomSymbolicDist_{name}")
             return _CustomSymbolicDist(
                 name,
                 *dist_params,
-                class_name=name,
```

Comment:
Wait, was old behavior to use the RV name as the class name?

Reply:
We were doing `"CustomDist_{class_name}", so yes, if a user created one in a model context it was `CustomDIst_mu` for example. Pretty meh

Reply:
No concern, just checking my understanding